.eslintrc.js

```diff
@@ -0,0 +1,205 @@
+        "@typescript-eslint/member-delimiter-style": [
+            "error",
+            {
+                "multiline": {
+                    "delimiter": "semi",
+                    "requireLast": true
+                },
+                "singleline": {
+                    "delimiter": "semi",
+                    "requireLast": false
+                }
+            }
+        ],
+        "@typescript-eslint/member-ordering": "error",
+        "@typescript-eslint/naming-convention": [
+            "error",
+            {
+                "selector": ["default"],
+                "format": ["camelCase", "PascalCase", "UPPER_CASE"],
+                "leadingUnderscore": "allow",
+                "trailingUnderscore": "allow"
+            },
+            {
+                "selector": ["classProperty", "classMethod"],
+                "modifiers": ["private"],
+                "format": ["camelCase"],
+                "leadingUnderscore": "require"
+            },
+            {
+                "selector": ["classProperty"],
+                "modifiers": ["readonly"],
+                "format": ["UPPER_CASE"],
+                "trailingUnderscore": "allow"
+            }
+        ],
+        "@typescript-eslint/no-empty-function": "off",
+        "@typescript-eslint/no-empty-interface": "error",
+        "@typescript-eslint/no-inferrable-types": [
+            "error",
+            {
+                "ignoreParameters": true
+            }
+        ],
+        "@typescript-eslint/no-misused-new": "error",
+        "@typescript-eslint/no-non-null-assertion": "error",
+        "@typescript-eslint/no-shadow": [
+            "error",
+            {
+                "hoist": "all"
+            }
+        ],
+        "@typescript-eslint/no-unused-expressions": "error",
+        "@typescript-eslint/no-use-before-define": "error",
+        "@typescript-eslint/prefer-function-type": "error",
+        "@typescript-eslint/quotes": [
+            "error",
+            "single",
+            {
+                "avoidEscape": true
+            }
+        ],
+        "@typescript-eslint/semi": [
+            "error",
+            "always"
+        ],
+        "@typescript-eslint/type-annotation-spacing": "error",
+        "@typescript-eslint/unified-signatures": "error",
+        "arrow-body-style": "error",
+        "brace-style": [
+            "error",
+            "1tbs"
+        ],
+        "capitalized-comments": [
+            "error",
+            "never"
+        ],
+        "constructor-super": "error",
+        "curly": "error",
+        "eol-last": "error",
+        "eqeqeq": [
+            "error",
+            "smart"
+        ],
+        "guard-for-in": "error",
+        "id-blacklist": "off",
+        "id-match": "off",
+        "import/no-deprecated": "warn",
+        "jsdoc/no-types": "off",
+        "max-len": [
+            "error",
+            {
+                "code": 200
+            }
+        ],
```

Comment:
I don't how how the whole translation process looks like but it seems there is some bug here. It should be `max-lenght`. So was thats manually copied by you @kilchenmann or this translation tool bug?

Another question here is, if that rule, and maybe others here (which I can recognise from old TSLing) are applied, because this one for instance isn't in the rules directory:
https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin/docs/rules

Reply:
No, I think `max-len` is correct? <https://eslint.org/docs/rules/max-len>

Reply:
This was the rule in `tslint.json`:

```json
"max-line-length": [
    true,
    200
],
```

Reply:
Ok, thanks, then it's me this time being blind ;)